### PR TITLE
[solidjs-pod] 2. Add framework theme

### LINKED ISSUE #271: [solidjs-pod] 2. Add framework theme
# Add theme for current framework

## Background

Each framework.dev site is built from the same codebase and it utilizes specific theme to provide each framework a different look and feel.

## Acceptance

- [x] Define the colour pallete for the technology (either by contacting the designers, or by researching on the technology site)
- [x] Create a new color schema in "@framework/packages/system/src/themes"
- [x] Import the light theme in "@framework/package/system/src/themes/themes.css.ts"
- [x] Import the dark theme in "@framework/package/system/src/themes/themes.css.ts"
- [x] Add theme preview in storybook "@framework/package/system/.storybook/preview.js"

### PR DESCRIPTION
**Pull request by kodejuice**
Should close #271 

## Type of change

- [x] Content addition
- [ ] Bug fix
- [ ] Behavior change

## Summary of change

Add SolidJS theme for site

### CODE REVIEW

File: packages/system/src/themes/solidjs-color-scheme.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { ColorPalette } from "./theme-from-palette"
+
+export const solidJSColorPalette: ColorPalette = {
+	highlight: "#2c4f7c",
+	logo: "#457ABF",
+	primary100: "#ffffff",
```
Comment (vyktoremario) [inline packages/system/src/themes/solidjs-color-scheme.ts:6]: Just for clarity. How was this colour palette derived? Were you provided with this or its random?

File: packages/system/src/components/logo.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ export function Logo({ className, siteName, stretched }: LogoProps) {
 		react: <ReactFrameworkIcon size="full" stretched={stretched} />,
 		angular: <AngularFrameworkIcon size="full" stretched={stretched} />,
 		vue: <VueFrameworkIcon size="full" stretched={stretched} />,
+		// solidjs: <VueFrameworkIcon size="full" stretched={stretched} />,
```
Comment (vyktoremario) [inline packages/system/src/components/logo.tsx:19]: This can be removed.

**Review by vyktoremario — APPROVED** [review]
Approved with a comment and a question🙂

File: packages/system/src/themes/solidjs-color-scheme.ts (hunk 1 above)
Comment (kodejuice) [reply packages/system/src/themes/solidjs-color-scheme.ts:6]: extracted the color palette from the solidJS website, using this tool https://www.colorcombos.com

But i think we'd probably need to get these colors using the same method that was used to generated the others.

Comment (vyktoremario) [reply packages/system/src/themes/solidjs-color-scheme.ts:6]: I agree. 
You can ask Dustin

File: packages/system/src/globals/global-constants.tsx [hunk 3]
```diff
@@ -31,4 +31,14 @@ export const FRAMEWORK_RESOURCES = (bigSize = false): LinkCardProps[] => [
 		backgroundColor: "#41B883",
 		bigSize,
 	},
+	{
+		title: "SolidJS Resources",
+		href: "https://solidjs.framework.dev/",
+		Icon(props) {
+			// return props;
```
Comment (hdJerry) [inline packages/system/src/globals/global-constants.tsx:38]: Do we need this comment?

Comment (kodejuice) [reply packages/system/src/globals/global-constants.tsx]: not at all

**Review by hdJerry — APPROVED** [review]
Shit ip

**Pull request merged**